src/StaticWebAssetsSdk/Tasks/ApplyCompressionNegotiation.cs

```diff
@@ -122,21 +135,44 @@ public override bool Execute()
                 foreach (var endpoint in endpoints)
                 {
                     Log.LogMessage(MessageImportance.Low, "    Adding endpoint '{0}'", endpoint.AssetFile);
-                }
-                foreach (var endpoint in endpoints)
-                {
+                    if (!HasVaryResponseHeaderWithAcceptEncoding(endpoint))
+                    {
+                        endpoint.ResponseHeaders = [
```

Comment:
If the endpoint already had a `Vary` header, but with different value, will we do the right thing by serving both values?